src/windows/CameraProxy.js

```diff
@@ -370,10 +395,11 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
             if (sensor !== null) {
                 sensor.addEventListener("orientationchanged", onOrientationChange);
             }
-            capturePreview.addEventListener('click', captureAction);
+            //capturePreview.addEventListener('click', captureAction);
```

Comment:
I will do another commit for u. Probably tomorrow morning now.

I am already signed up and sent my license off as contributed to other
cordova plug in projects
On 9 Jul 2015 7:41 pm, "Murat Sutunc" <notifications@github.com> wrote:

> In src/windows/CameraProxy.js
> <https://github.com/apache/cordova-plugin-camera/pull/106#discussion_r34288565>
> :
>
> > @@ -370,10 +395,11 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
> >              if (sensor !== null) {
> >                  sensor.addEventListener("orientationchanged", onOrientationChange);
> >              }
> > -            capturePreview.addEventListener('click', captureAction);
> > +            //capturePreview.addEventListener('click', captureAction);
>
> you can iterate with a new commit, I can merge them. Also before I can
> merge your changes, you need to sign Apache's Contributor License Agreement
> (can be done online): http://www.apache.org/licenses/#clas
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/cordova-plugin-camera/pull/106/files#r34288565>
> .
>